projects/dsp-ui/src/lib/viewer/values/value-error-state-matcher.ts

```diff
@@ -0,0 +1,9 @@
+import {FormControl, FormGroupDirective, NgForm} from '@angular/forms';
```

Comment:
Formatting: space missing between { } and the imports

Reply:
> Formatting: space missing between { } and the imports

I can certainly apply the rules we discussed to the new file `value-error-state-matcher`. Regarding the other files modified in this PR, I believe reformatting should be kept to a separate PR to avoid merge conflicts. what do you think?

Reply:
yes exactly, we reformat only new files, and the reformatting of the others will be done later on in a specific PR